src/server/tran_page_requests.hpp

```diff
@@ -56,19 +56,17 @@ enum class page_to_tran_request
   SEND_TO_PTS_LOG_PRIOR_LIST, /* response-less */
 };
 
-// requets from page server to page server to catchup
+// requests from page server to page server to catchup
 enum class follower_to_followee_request
```

Comment:
Nit
`ps_follower_to_ps_followed_request`
also, my browser spell checker says that `followee` is incorrect and that `followed` is correct

Reply:
My checker does too, but followee is used quite broadly and described in Wikis. I guess it's used in Instagram..